### PR TITLE
Sidepanel

### PR DESCRIPTION
**Pull request by davelahaye**
Added new selected and hover states for cards.

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -3,8 +3,8 @@ import fetch from 'isomorphic-fetch'
 import styled from 'styled-components'
 
 import PrimaryLayout from 'layouts/Primary'
-import CheckBox from 'atoms/icons/check_box_outline_blank-24px.svg'
-import CheckBoxSelected from 'atoms/icons/check_box-24px.svg'
+// import CheckBox from 'atoms/icons/check_box_outline_blank-24px.svg'
+// import CheckBoxSelected from 'atoms/icons/check_box-24px.svg'
```
Comment (brentmclark) [inline pages/index.js:7]: Can these be deleted instead of commented out?

File: pages/index.js [hunk 2]
```diff
@@ -133,9 +141,24 @@ const SearchResult = styled.div`
...
+
+const ResultNotSelected = styled.div`
   border: 2px solid #282828;
+  transition: .3s;
   box-shadow: 6px 6px 11px #282828, 
              -6px -6px 11px #3e3e3e;
+  &:hover {
+    border: 2px solid ${p => p.theme.color.tertiary};
+    transition: .3s;
+  }
```
Comment (brentmclark) [inline pages/index.js:161]: What are your thoughts on combining these two styled components into a single component.  They seem identical aside from the hover effect on the `NotSelected` variant.

Comment (davelahaye) [reply pages/index.js:161]: That was the only way I knew how to give a selected state based on your checkbox example.  I was excited to get that working after quite a bit of time hacking on it... even though I'm certain there is a better way.

File: pages/index.js (hunk 1 above)
Comment (davelahaye) [reply pages/index.js:7]: Yes.  You have found another one of my forgotten code children.  How do you manage all of the little changes like this when you are trying something out?  I got what I wanted to working and didn't scroll up to the imports area after that.  Do you have a process or checklist that you go through before a pull request?

**Pull request merged**